src/west/commands/build.py

```diff
@@ -188,6 +189,20 @@ def _setup_source_dir(self):
             source_dir = os.getcwd()
         self.source_dir = os.path.abspath(source_dir)
 
+    def _get_available_boards_str(self):
+        path = Path(os.getenv("ZEPHYR_BASE")+"/boards")
+        brds_paths = path.glob("**/Kconfig.board")
+        brds_db = {}
+        for brd in brds_paths:
+            try:
+                brds_db[brd.parent.parent.name].append(brd.parent.name)
+            except KeyError:
+                brds_db[brd.parent.parent.name] = [brd.parent.name, ]
+        brds_str = ""
+        for arch in brds_db.keys():
+            brds_str += "\r\n" + arch + ":\r\n\t\t" + "\r\n\t\t".join(brds_db[arch])
```

Comment:
Can you use print() instead of manually adding Windows newlines? West is a cross-platform program and needs to work on macOS and Linux, where just newlines are used.